docs/source/user_guide.rst

```diff
@@ -40,11 +40,16 @@ For example, we can define a random circuit in Qiskit and optimize it using the
    gates = ["cx", "cz", "cy", "swap", "x", "y", "z", "s", "sdg", "h"]
    num_qubits = 10
    raw_circuit = random_clifford_circuit(
-       num_qubits, gates=gates, num_gates=10 * num_qubits * num_qubits
+      num_qubits, gates=gates, num_gates=10 * num_qubits * num_qubits
    )
    compiled_circuit = ucc.compile(raw_circuit)
-   print(f"Number of multi-qubit gates in original circuit: {count_multi_qubit_gates_qiskit(raw_circuit)}")
-   print(f"Number of multi-qubit gates in compiled circuit: {count_multi_qubit_gates_qiskit(compiled_circuit)}")
+   print(
+      f"Number of multi-qubit gates in original circuit: {count_multi_qubit_gates_qiskit(raw_circuit)}"
+   )
+   print(
+      f"Number of multi-qubit gates in compiled circuit: {count_multi_qubit_gates_qiskit(compiled_circuit)}"
+   )
```

Comment:
nit: Indentation with 3 spaces 😱.

Reply:
Yikes --- so embarrassing 😜 !!

Reply:
Actually, double-embarrass perhaps. Looking again at existing `rst`, it looks like the code within codeblocks is currently indented with 3 spaces.

https://github.com/unitaryfund/ucc/blob/d92c3419a7d10dcdcb1bf29ef37662b7dc9dd3db/docs/source/user_guide.rst?plain=1#L12-L15

Reply:
Is that a standard `rst` thing?

Reply:
The example in the [docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block) has that pattern, but just testing, looks like as long as the indentation is consistent, the number of spaces doesn't matter.

If that's the case, I'll leave as is for now (where its 3 spaces throughout that file)